tools/tagger/tagger.js

```diff
@@ -9,7 +9,7 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
-import getTaxonomy from '../taxonomy.js';
+import getTaxonomy from '../../hub/taxonomy.js';
```

Comment:
`taxonomy.js` can be moved out of `hub/` if it will not be used in the project; for the moment I'll keep it in.